packages/commands/src/components/command-menu.js

```diff
@@ -50,6 +50,7 @@ function CommandMenuLoader( { name, search, hook, setLoader, close } ) {
 				<Command.Item
 					key={ command.name }
 					value={ command.searchLabel ?? command.label }
+					keywords={ command.keywords }
```

Comment:
Oh I see this is built-in behaviour in https://www.npmjs.com/package/cmdk

Nice!